frame/staking/src/pallet/impls.rs

```diff
@@ -793,6 +795,12 @@ impl<T: Config> Pallet<T> {
 			nominators_taken
 		);
 
+		// `all_voters` is sorted by vote weight, so we use the last element to set the current
+		// minimum active bond. Note that depending on the `T::VoterList` list, the sorting may be
+		// inconsistent as the sorting may be lazy.
+		let min_active_bond: T::CurrencyBalance = all_voters.last().map_or(0, |v| v.1).into();
```

Comment:
I wonder if we should iterate over `all_voters` and pick the min bond to account for inconsistencies in lazily sorted `T::VoterList` like bags-list.

Reply:
We are already iterating over all_voters right? Is there a reason not to keep track of the minimum active bond in that loop?

Reply:
We are iterating over `sorted_voters` to be accurate.